packages/dts-generator/lib/phases/json-to-ast.js

```diff
@@ -280,6 +280,13 @@ function buildClass(ui5Class) {
   return astNode;
 }
 
+const ui5ExtendsToArray = (ui5Extends) => {
+	if ( Array.isArray(ui5Extends) ) {
```

Comment:
This project uses `lodash` so lets use _.isArray and _.clone.
The second one is also clearer than `slice` as the verb is closer to the intent.

Reply:
Done